input/pagecontent/background.md

```diff
@@ -0,0 +1,14 @@
+### NCPI Background Information
+#### 3.1.1 - Overview
+TODO: Overview - Single Paragraph
+
+![NCPI Cloud](ncpi-cloud.png)
+
+#### 3.1.2 - Scientific/Platform Problem Statement
+TBD
+
+#### 3.1.3 - FHIR Relevance
+TBD
+
+#### 3.1.4 - IG Scope
+TBD
```

Comment:
I generated an IG off of your branch and these subsections are located next to the figure above (meaning there are two columns with the figure on the left column and the rest of the subections on the right column). Plus, it seems the section numbering next to the hash sign is not needed since the publisher auto-assigns the header numbers. So, now it's like:

3.1.1 3.1.1 - Overview